pointerevents/pointerevent_support.js

```diff
@@ -185,3 +193,49 @@ function rPointerCapture(e) {
     catch(e) {
     }
 }
+
+var globalPointerEventTest = null;
+var expectedPointerType = null;
+
+var HOVERABLE_POINTERS = ['mouse', 'pen'];
+
+function MultiPointerTypeTest(testName, types) {
+    this.testName = testName;
+    this.types = types;
+    this.currentTypeIndex = 0;
+    this.createNextTest();
+}
+
+MultiPointerTypeTest.prototype.skip = function() {
+    this.createNextTest().timeout();
```

Comment:
I thought that "no active test" condition applied only at event loop boundaries, no?  So this is just the pattern than whenever you have multiple async_tests in testharness.js tests, you always create the next one before terminating the last one?  That's really weird IMHO.  Another way you could solve this to get that order is like:
```javascript
  setTimeout(this.currentTest.timeout, 0);
  this.createNextTest();
```